src/lib.rs

```diff
@@ -44,12 +44,7 @@ extern crate packed_simd;
 extern crate rand;
 extern crate clear_on_drop;
 extern crate byteorder;
-
-// The `Digest` trait is implemented using `generic_array`, so we need it
-// too. Hopefully we can eliminate `generic_array` from `Digest` once const
-// generics land.
-extern crate digest;
-extern crate generic_array;
+pub extern crate digest;
```

Comment:
By making this `pub`, `ed25519-dalek` can depend on it rather than having its own `extern digest` declaration, and we can be sure the two are in lockstep version-wise.